js/schematic.js

```diff
@@ -13,6 +13,7 @@ var signal = d3.dispatch(
         .connect(signal)
   , atlas = Atlas()
         .tooltipContent(tooltipContent)
+        .connect(signal)
```

Comment:
haha, should've waited for the next commit!  I forgot that atlas wasn't connected at all!